cvat/apps/webhooks/signals.py

```diff
@@ -203,15 +199,15 @@ def resource_create(sender, instance, created, **kwargs):
     batch_add_to_queue(filtered_webhooks, data)
 
 
-@receiver(post_delete, sender=Project, dispatch_uid="project:delete")
-@receiver(post_delete, sender=Task, dispatch_uid="task:delete")
-@receiver(post_delete, sender=Job, dispatch_uid="job:delete")
-@receiver(post_delete, sender=Label, dispatch_uid="label:delete")
-@receiver(post_delete, sender=Issue, dispatch_uid="issue:delete")
-@receiver(post_delete, sender=Comment, dispatch_uid="comment:delete")
-@receiver(post_delete, sender=Organization, dispatch_uid="organization:delete")
-@receiver(post_delete, sender=Invitation, dispatch_uid="invitation:delete")
-@receiver(post_delete, sender=Membership, dispatch_uid="membership:delete")
+@receiver(pre_delete, sender=Project, dispatch_uid="project:delete")
```

Comment:
dispatch_uid should be unique. Probably better to add __name__ to the name. Please look how it is used in other places.